README.md

```diff
@@ -34,6 +34,17 @@ The image recognizes following environment variables that you can set during ini
 |  `MYSQL_DATABASE`      | Database name                             |
 |  `MYSQL_ROOT_PASSWORD` | Password for the root user (optional)     |
 
+Following environment variables influence MySQL configuration file. They are all optional.
+
+|    Variable name                |    Description                                                    |    Default
+| :------------------------------ | ----------------------------------------------------------------- | -------------------------------
+|  `MYSQL_LOWER_CASE_TABLE_NAMES` | Sets how the table names are stored and compared                  |  0
+|  `MYSQL_DEFAULT_STORAGE_ENGINE` | The default storage engine (table type)                           |  InnoDB
```

Comment:
this might break the VOLUME data if you update it for Pod. If the VOLUME already exists and it use ISAM and you change it to InnoDB the mysql won't start. Also in latest MySQL they are going to drop MyIsam so I think we should remove this option (dangerous and potentially going away in near future)